roles/alexandria-v2/templates/environment.rb.j2

```diff
@@ -0,0 +1,7 @@
+# Load the Rails application.
+require File.expand_path('../application', __FILE__)
+
+Rails.env = ENV['RAILS_ENV'] || '{{ rails_env }}'
```

Comment:
Why is this overwriting config/environment.rb from the main codebase? Changes to this file should happen in alexandria-v2, otherwise they will be overwritten by the next cap deploy.